components/resources/demo/shared/build.gradle.kts

```diff
@@ -50,21 +50,32 @@ kotlin {
         }
     }
 
+    applyDefaultHierarchyTemplate()
     sourceSets {
         all {
             languageSettings {
                 optIn("org.jetbrains.compose.resources.ExperimentalResourceApi")
             }
         }
+        val desktopMain by getting
+        val wasmJsMain by getting
+
         commonMain.dependencies {
             implementation(compose.runtime)
             implementation(compose.material3)
             implementation(project(":resources:library"))
         }
-        val desktopMain by getting
         desktopMain.dependencies {
             implementation(compose.desktop.common)
         }
+
+        val nonAndroidMain by creating {
```

Comment:
We agreed to call it `jbMain` (renaming for core repo in todo)

Reply:
it is the demo app. don't think it matters

Reply:
I wanted to make an attention on the fact that Android doesn't support SVGs